tests/next_tests/integration_tests/regression_tests/ffront_tests/test_offset_dimensions_names.py

```diff
@@ -0,0 +1,60 @@
+# GT4Py - GridTools Framework
```

Comment:
I created this new test category to distinguish backend bug fixes from general features. Maybe it should be one level up?